Live-Snap/Live-Snap/Models/ImageManager.swift

```diff
@@ -32,7 +32,36 @@ class ImageManager {
     
     func requestPhotoLibraryPermission(completion: @escaping (Bool) -> ()) {
         PHPhotoLibrary.requestAuthorization { (status: PHAuthorizationStatus) in
-            completion(status == .denied || status == .restricted)
+            completion(!(status == .denied || status == .restricted))
+        }
+    }
+    
+    func photoLibraryPermissionHandler() {
```

Comment:
rename to better explain what this does: checkPhotoLibraryPermissions() would probably be better